src/cn-toast.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+  'use strict';
+
+  angular.module('cn.ui')
+      .controller('toastController', [
+        '$rootScope', 'toaster',
+        function($rootScope, toaster) {
+          const defaults = {
+            timeout: 5000,
+            toasterId: 'simple',
+            body: 'simple-toast',
+            bodyOutputType: 'directive',
+            tapToDismiss: false,
+            closeHtml: '<a>Dismiss</a>',
+            directiveData: {
+              type: 'success'
+            }
+          };
+
+          $rootScope.$on("citizenNet:toastEvent", function(event, options) {
```

Comment:
`options = {}`

Set default value so we don't run into problems with my suggestion below